lib/pbio/drv/uart/uart_stm32f0.c

```diff
@@ -265,6 +272,8 @@ PROCESS_THREAD(pbdrv_uart_process, ev, data) {
         uart->initialized = true;
     }
 
+    pbdrv_init_busy_down();
```

Comment:
I don't see any yields between busy up and down, so it doesn't seem like it is needed here.